mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.purgeOrder = async (req, res) => {
+  try {
+    const result = await Order.findOneAndRemove({ _id: req.params.orderId });
+    if (!result) {
+      return res.status(404).json({ message: "Order not located" });
+    }
+    res.status(200).json({ message: "Order eradicated successfully" });
+  } catch (err) {
+    res.status(500).json({ message: errorHandler(err) });
```

Comment:
In the error response, it would be better to include the error details along with the error message. This will provide more information to the client about the error. 

Instead of just `res.status(500).json({ message: errorHandler(err) });`, consider `res.status(500).json({ message: errorHandler(err), error: err });`. This will include the actual error details in the response.